src/lib/offline-api/index.ts

```diff
@@ -137,9 +150,11 @@ class OfflineAPI {
   private requestBatches: RequestBatch[] = []
   private contentTypeValidators: ContentTypePayloadValidator[] = []
   private locales: string[] = []
+  private modifiedTags: Map<String, Tag> = null
 
-  constructor (contentTypes: Map<String, ContentType> = new Map(), entries: Entry[] = [], locales: string[], editorInterfacesByContentType: Map<String, EditorInterfaces> = new Map<String, EditorInterfaces>()) {
+  constructor (contentTypes: Map<String, ContentType> = new Map(), entries: Entry[] = [], locales: string[], editorInterfacesByContentType: Map<String, EditorInterfaces> = new Map<String, EditorInterfaces>(), tags: Map<String, Tag> = null) {
```

Comment:
Shouldn't we also initialize the `tags` map to an empty map instead of `null`?

Also I think it's time to refactor this to use an object in the constructor instead of separate positional arguments.

Reply:
Yes, I changed it to a map initialization. I will refactor the constructor in a different pr!